index.html

```diff
@@ -1,13 +1,232 @@
+                <p class="bio d-text">
+                    I'm a software developer! I can help you build a product, feature or website. Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don't hesitate
+                    to contact me.
+                </p>
+                <span class="light-purple letsconnect">LET'S CONNECT</span>
+                <ul class="icons">
+                        <li>
+                          <a href="YOUR SOCIAL MEDIA HANDLE">
+                            <img src="./assets/twitter.png" class="icon" alt="twitter icon" />
+                          </a>
+                        </li>
+                        <li>
+                          <a href="YOUR SOCIAL MEDIA HANDLE">
+                            <img src="./assets/Linkedin icon.png" class="icon" alt="twitter icon" />
+                          </a>
+                        </li>
+                        <li>
+                          <a href="YOUR SOCIAL MEDIA HANDLE">
+                            <img src="./assets/medium.png" class="icon" alt="twitter icon" />
+                          </a>
+                        </li>
+                        <li>
+                          <a href="YOUR SOCIAL MEDIA HANDLE">
+                            <img src="./assets/github.png" class="icon" alt="twitter icon" />
+                          </a>
+                        </li>
+                        <li>
+                          <a href="YOUR SOCIAL MEDIA HANDLE">
+                            <img src="./assets/angellist.png" class="icon" alt="twitter icon" />
+                          </a>
+                        </li>
+                      </ul>
+            </div>
+        </section>
+      
+    <section class="portfolio test">
+      <div class="project">
+          <img src="./assets/tonic.png" alt="" class="project-preview">
+
+          <div class="project-info">
+              <h2 class="h-text project-name">Tonic</h2>
+                <ul class="skills-list">
+                  <li class="checkbox"><span class="info-text">CANOPY</span></li>
+                  <li class="checkbox"><i class="ri-checkbox-blank-circle-fill dots"></i>
+                    <span class="info-text">Back End Dev</span></li>
+                  <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                    <span class="info-text">2015</span></li>
+                </ul>
+              
+              <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <ul class="tech-list">
+              <li class="project-tech">html</li>
+              <li class="project-tech">css</li>
+              <li class="project-tech">javascript</li>
+            </ul>
+              <button class="outlined-button">See Project</button>
+          
+          </div>
+      </div>
+
```

Comment:
- [ ] For each of your project cards, kindly check to make sure that your spacing units (whether `padding`, `margin` or `gap`) match the figma design. Also, `font-sizes`, `background-color`, and `colors`. It is important that you take those specific properties from the template given you. Kindly treat it as your clients requirement which you should try as much as possible to deliver.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/219874949-cc3a8433-6111-4876-8e4d-14a515f77c53.png)|![image](https://user-images.githubusercontent.com/99715640/219874968-17974ece-467d-429b-aafa-2578b37aa2b6.png)|

- [ ] Also, for all you buttons you seem to forget to add some interactions like `hover` and `active` states. Kindly add them as specified by the Figma design. Here is a copy of the design palette :point_down: 

![image](https://user-images.githubusercontent.com/99715640/219875757-6f27d9e0-6513-4d3d-8300-3d7308a6509a.png)